### DISCUSSION TITLE
eshell aliases file not created

### DISCUSSION THREAD

**Discussion by doolio** [emacscollective/no-littering#234] [Support] [answered]
Running latest `no-littering` at time of writing namely commit [dcc7af4](https://github.com/emacscollective/no-littering/commit/dcc7af41cc6b48d446c0a76bae029898f82cc2e3).

I wanted to create some eshell aliases. I managed to create them but they did not persist i.e. they were not saved to `eshell-aliases-file`. When I lookup the value of this variable I see the following:

```
Value
"/home/doolio/.emacs.d/etc/eshell/aliases"

Original Value
"/home/doolio/.emacs.d/var/eshell/alias"
```

which I think is fine on account of #135 and #138. However, it seems #138 has been [undone](https://github.com/emacscollective/no-littering/blob/dcc7af41cc6b48d446c0a76bae029898f82cc2e3/no-littering.el#L134) but I can't find the reason why. Was it perhaps forgotten when the creation of specific directories was [regrouped](https://github.com/emacscollective/no-littering/blob/dcc7af41cc6b48d446c0a76bae029898f82cc2e3/no-littering.el#L187)? If so I can submit a PR to have it created again.

Thanks for your time.

**Comment by tarsius** [comment, marked as answer]
Fixed in 5bc4d5e4980c261d2c1c470f92896e816fb6ada7.

Sorry this took so long.
